$NetBSD: patch-aq,v 1.7 2007/06/12 21:43:30 heinz Exp $

  Purely changes for pkgsrc:

	- Change paths to those used by pkgsrc
	- Notes about search paths set by pkgsrc

--- README.orig	2007-06-08 14:56:25.000000000 +0200
+++ README
@@ -99,25 +99,25 @@ Mail::SpamAssassin::Conf class (run the 
 the standard defaults that people use.  There is an explanation of all the
 default locations that SpamAssassin will look at the end.
 
-  - /usr/share/spamassassin/*.cf:
+  - @@PREFIX@@/share/spamassassin/*.cf:
 
 	Distributed configuration files, with all defaults.  Do not modify
 	these, as they are overwritten when you upgrade.
 
-  - /var/lib/spamassassin/*/*.cf:
+  - @VARBASE@/spamassassin/*/*.cf:
 
         Local state directory; updated rulesets, overriding the
         distributed configuration files, downloaded using "sa-update". Do
         not modify these, as they are overwritten when you run
         "sa-update".
 
-  - /etc/mail/spamassassin/*.cf:
+  - @PKG_SYSCONFDIR@/*.cf:
 
   	Site config files, for system admins to create, modify, and
 	add local rules and scores to.  Modifications here will be
 	appended to the config loaded from the above directory.
 
-  - /etc/mail/spamassassin/*.pre:
+  - @PKG_SYSCONFDIR@/*.pre:
 
         Plugin control files, installed from the distribution. These are
         used to control what plugins are loaded.  Modifications here will
@@ -126,22 +126,22 @@ default locations that SpamAssassin will
         
         You want to modify these files if you want to load additional
         plugins, or inhibit loading a plugin that is enabled by default.
-        If the files exist in /etc/mail/spamassassin, they will not
+        If the files exist in @PKG_SYSCONFDIR@, they will not
         be overwritten during future installs.
 
-  - /usr/share/spamassassin/user_prefs.template:
+  - @@PREFIX@@/share/spamassassin/user_prefs.template:
 
 	Distributed default user preferences. Do not modify this, as it is
 	overwritten when you upgrade.
 
-  - /etc/mail/spamassassin/user_prefs.template:
+  - @PKG_SYSCONFDIR@/user_prefs.template:
 
 	Default user preferences, for system admins to create, modify, and
 	set defaults for users' preferences files.  Takes precedence over
 	the above prefs file, if it exists.
 
         Do not put system-wide settings in here; put them in a file in the
-        "/etc/mail/spamassassin" directory ending in ".cf". This file is
+        "@PKG_SYSCONFDIR@" directory ending in ".cf". This file is
         just a template, which will be copied to a user's home directory
         for them to change.
 
@@ -195,6 +195,12 @@ These are:
   /opt/$DIR     /opt/$DIR/share/spamassassin   /etc/opt/mail/spamassassin
   $DIR          $DIR/share/spamassassin        $DIR/etc/mail/spamassassin
 
+(Pkgsrc note: the pkgsrc build process used the following path names
+  __prefix__:          @@PREFIX@@
+  __def_rules_dir__:   @@PREFIX@@/share/spamassassin
+  __local_rules_dir__: @PKG_SYSCONFDIR@
+)
+
 The files themselves are then looked for in these paths:
 
   - Distributed Configuration Files
@@ -234,6 +240,11 @@ ruleset:
   /opt/$DIR     /var/opt/spamassassin/__version__
   $DIR          $DIR/var/spamassassin/__version__
 
+(Pkgsrc note: the pkgsrc build process used the following path names
+  __prefix__:          @@PREFIX@@
+  __local_state_dir__: @VARBASE@/spamassassin/__version__
+)
+
 This is normally written to by the "sa-update" script.  "__version__" is
 replaced by a representation of the version number, so that multiple
 versions of SpamAssassin will not interfere with each other's rulesets.
